IndicoV2/IndicoClientExtensions.cs

```diff
@@ -102,12 +101,5 @@ public static class IndicoClientExtensions
             new DataSetClient(new DataSetsV1ClientAdapter(indicoClient.LegacyClient),
                 indicoClient.IndicoStrawberryShakeClient.DataSets(), indicoClient.Storage());
 
-        /// <summary>
-        /// Gets <seealso cref="IGraphQLRequestClient"/>
-        /// </summary>
-        /// <param name="indicoClient">Instance of <seealso cref="IndicoClient"/></param>
-        /// <returns>Instance of <seealso cref="IGraphQLRequestClient"/></returns>
-        public static IGraphQLRequestClient GraphQLRequest(this IndicoClient indicoClient) =>
```

Comment:
What happens with current consumers who are using this call?

Reply:
The call itself got moved in the IndicoClient.cs. Instead of `client.GraphQLRequest().Call(query, operationName, variables)` it's `client.GraphQLRequest(query, operationName, variables).Call()`